salt/haproxy/config/haproxy.cfg.jinja

```diff
@@ -117,6 +116,24 @@ frontend main
     bind :::80
     bind 127.0.0.1:19001  # This is our TLS socket.
 
+    # Define a stick table for all services
+    stick-table type ip size 100k expire 30s store http_req_rate(10s)
```

Comment:
[Docs](https://www.haproxy.com/documentation/haproxy-configuration-manual/latest/#stick-table) say that `type ip` only stores ipv4 addresses, do we need a separate table to ensure we also rate limit ipv6?

Reply:
this is what I get for splitting my work into two branches after I've done it in one 🙃

Reply:
okay,[587e709](https://github.com/python/psf-salt/pull/509/commits/587e709e3a7cb2873555bd7ba49152fe392a85c8)